crates/neon/src/types_impl/buffer/mod.rs

```diff
@@ -47,7 +47,7 @@ pub use types::Binary;
 /// ```
 ///
 /// [typed-arrays]: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Typed_arrays
-pub trait TypedArray: private::Sealed {
+pub trait TypedArray: private::Sealed + Value {
```

Comment:
We could drop `Sealed` here since `Value` is already sealed with `ValueInternal`.